style.css

```diff
@@ -0,0 +1,80 @@
+html {
+    scroll-behavior: smooth;
+}
```

Comment:
Let's add a blank lines after each selector's block. It will make this file more readable